### PR TITLE
Change flying-robot-frame to motor-2

### PR DESCRIPTION
**Pull request by lambbacons**
Changing to motor-2 instead of motor-3 opens up flying robots much earlier in the progression, so it's not a late game technology. Many real drones use DC brushless motors anyways. 

Maybe its ingredients should be two motors instead of one like flying-robot-frame-2

I don't know why it's adding a new line at the end. Sorry.

### CODE REVIEW

**Comment by Repofme1** [comment]
Hm.  To be honest, I kind of like robots as a later-game technology.  The combination of high power, energy, computing ability, sensors, etc. in a small, very mass-limited machine doesn't really sound like mid-game to me.  Plus, the motor 3 is actually the brushless motor in XM, and has been for a long time.  I switched it with the induction motor, which used to be tier 3, back when someone pointed out the technological precedence of the induction motor.

There are also the resulting technical issues from this change, like switching out the technology prerequisites, re-balancing things, etc.  Investigating something so high up in the tech tree, when I have been focusing on foundation work recently, would almost certainly get me lost in speculation.  I do have ideas for what to do with robots, for instance, I don't want them to be a super late game technology, more like the transition between middle and late.  I really like the idea of construction robots, and slightly less the idea of short-range logistics bots to resupply the player, but not as much the base where almost everything goes through logistics bots.  I would like to make a way to more gradually introduce robots, but that's a topic for later.

Long story short, I'm not even near reaching this point in upgrading the mod.  All in good time.

**Comment by lambbacons** [comment]
Thanks a lot for the explanation! I'm sorry to distract from things; I just wanted to help however I can. I'm also new to github, so I'm learning... 

And thanks for all the hard work on this mod. It's my definitely my favorite and I love where these changes are headed.

**Comment by Repofme1** [comment]
I'm glad you like the mod, and I appreciate the offer of help.  I like to accept as much of this as I can, and it makes me happy that people think about these things, I simply can't easily reconcile it all.  It is unfortunate that my 'master plan', so to speak, is so hard to get out of my head!  By that I mean it is difficult to explain to people, and even hides itself from me; it mainly surfaces when a suggestion or bug that I see prompts a train of thought about how that should be fixed.
